block-storage-attacher/Makefile

```diff
@@ -80,5 +79,5 @@ build-systemutil:
 
 .PHONY: oss
 oss:
-	go get github.ibm.com/alchemy-containers/armada-opensource-lib@v0.4
-	go run github.ibm.com/alchemy-containers/armada-opensource-lib/cmd/makeoss ${OSS_FILES}
+	#go get github.ibm.com/alchemy-containers/armada-opensource-lib@v0.4
```

Comment:
Dont remove the `oss` task. Please revert this.

Reply:
This is also from github.ibm.com we will land into same issue

Reply:
reverted